test/unit/nedap/utils/test/api.cljc

```diff
@@ -129,6 +130,14 @@
                   :from 1
                   :to 3)))
 
+  (testing ":with `clojure.test/=`"
```

Comment:
In addition to this test, we should have a test that contrasts `=`, `meta=` and in a future `match`, via `are`

Reply:
That's not possible afaik. Because of it will error on macro expansion. I might be able to extend that test though

Reply:
Gave this another thought; this is already covered by `:to failures`, don't you think?

Reply:
Adding it there sgtm, thanks!